### DISCUSSION TITLE
HELP!

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#63] [Domande & Rispsote]
Aiuto!

Sto traducendo il file beckett.dlg, sono bloccato per una frase alquanto dubbia per me.
Anzi mi sembra un pò stramba, ma se sapessi il contesto preciso almeno...
non riesco ad andare avanti se non sbriglio sto nodo.

Il pezzo della frase in questione che mi desta dubbi è questa :

" To the batty signal! "

La frase completa e la linea 143 del file beckett.dlg

143	Who stole it?            A supervillain must have stolen it! To the batty signal!

Ho provato a guardare nelle 2 traduzioni disponibili e la frase  " To the batty signal! "

ho trovato questa traduzione :

1) -  Al bat-segnale!

2) -  Deve essere proprio pazzo!

dopo averle lette lo confesso sono ancora più perplesso.

perchè sia una che l'altra qualcosa di giusto c'è l'hanno.

"batty" può significare (pazzo, irrazionale, fuori di testa, inteso in forma leggera, etc...)
ma la frase tradotta purtroppo non dice proprio quello...

Mmmm, il riferimento a batman potrebbe essere giusto se chi parla lo sta facendo in slang,
e se sta usando un tono estremamente ironico sarcastico.
(non posso saperlo. appunto, mi manca il contesto...)

ma ho ancora qualche piccolo dubbio, ecco perchè mi servirebbe il contesto da chi il gioco lo ha giocato e magari sà di cosa si parla nel contesto della frase

quella frase potrebbe anche significare altro, ma senza contesto preciso si va a sbattere alla cieca, quindi mi domando se sono su una barca, o sul molo di un porto...
Perche potrebbe centrarci un tipo di segnalatore montato su barche o porti, non ricordo esattamente...

Qualcuno sa qualcosa riguardo al contesto? opinioni sulla frase?

Thanks

**Comment by NarakuITA** [comment]
Sto chiedendo aiuto al Forum di traduzioni. Vediamo se qualcuno ci aiuta.

**Comment by crazyLillo** [comment]
Grazie Mille!

comunque a me interessava il contesto da parte di chi ci ha giocato e si ricorda quel pezzo, magari.
ecco perchè domandavo se erano in porto o su una barca, per il flebile motivo che ho spiegato nel messaggio precedente. (ma ho i miei dubbi che sia così).

Ad ogni modo credo che il riferimento a batman sia giusto, perchè la frase penso sia detta in modo molto ironico e sarcastico.

del tipo :

Nota: "non è la traduzione della frase!"

"Ah! un supercrminale l'ha rubata?, in questo caso allora chiamiamo batmam che forse solo lui la ritrova..."

( è più o meno quello il senso dela frase a mio modesto parere...)

Sono convinto che sia così, quindi lascerò la frase "To the batty signal!"--- "Al Bat-segnale!".

se eventualmente esce fuori qualcos'altro, ci metto un attimo a corregere. ma non credo succederà.

**Reply by NarakuITA** [reply]
Per adesso, nessuno mi a saputo dire. Io purtroppo il gioco non l'ho mai giocato.

**Reply by crazyLillo** [reply]
Grazie lo stesso, ho risolto andandomi a guardare i vari video sul web di chi ci ha giocato.
Sono in un museo!
Quindi il mio dubbio è svanito.
Poi ho scoperto che beckett è il sarcasmo fatto persona anzi lupo...
Quindi il riferimento a batman è giusto!

Come non ci hai mai giocato? ho visto che hai postato immagini del gioco...
